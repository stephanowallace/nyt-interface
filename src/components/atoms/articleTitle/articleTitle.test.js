import React from 'react';
import { shallow } from 'enzyme';
import ArticleTitle from '.';

describe('Article title', () => {
  it('renders default state', () => {
    const component = shallow(<ArticleTitle>test</ArticleTitle>);
    expect(component).toMatchSnapshot();
  });

  it('renders loader', () => {
    const component = shallow(<ArticleTitle loading>test</ArticleTitle>);
    expect(component).toMatchSnapshot();
  });

  it('renders child in an h2 tag', () => {
    const component = shallow(<ArticleTitle>test</ArticleTitle>);
    expect(component.find('h2').text()).toBe('test');
  });
});
