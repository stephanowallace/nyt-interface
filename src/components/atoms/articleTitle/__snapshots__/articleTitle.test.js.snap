// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Article title renders default state 1`] = `
<h2
  className="title "
>
  test
</h2>
`;

exports[`Article title renders loader 1`] = `
<ContentLoader
  className="title"
  height="26px"
  width="300px"
/>
`;
